apps/api/src/persistence/database/arango-database.ts

```diff
@@ -55,11 +55,12 @@ export class ArangoDatabase {
             throw new InternalError(`Arango cannot fetchById with invalid id: ${id}`);
         }
 
-        if (!isArangoCollectionId(collectionName)) {
-            throw new InternalError(
-                `Arango cannot fetchById from an invalid collection: ${collectionName}`
-            );
-        }
+        // TODO Do we need this?
```

Comment:
We need to loosen our grip a bit on our implementation so that Arango doesn't need to be statically coupled to the allowed collection IDs. If we really want to check if the collection is a valid one
- we can always check if exists, given that we will create all required collections at bootstrap
- these collections should be registered dynamically via annotations